src/tabs/tabs.js

```diff
@@ -61,10 +61,56 @@
     MDL_JS_RIPPLE_EFFECT: 'mdl-js-ripple-effect',
     MDL_RIPPLE_CONTAINER: 'mdl-tabs__ripple-container',
     MDL_RIPPLE: 'mdl-ripple',
-    MDL_JS_RIPPLE_EFFECT_IGNORE_EVENTS: 'mdl-js-ripple-effect--ignore-events'
+    MDL_JS_RIPPLE_EFFECT_IGNORE_EVENTS: 'mdl-js-ripple-effect--ignore-events',
+    TAB_BAR_CLASS: 'mdl-tabs__tab-bar'
   };
 
   /**
+     * Keycodes, for code readability.
+     *
+     * @enum {number}
+     * @private
+     */
+  MaterialTabs.prototype.Keycodes_ = {
+      LEFT_ARROW: 37,
+      RIGHT_ARROW: 39
+    };
+
+  /**
+     * Handles a keyboard navigation events on the drawer.
+     *
+     * @param {Event} evt The event that fired.
+     * @private
+     */
+  MaterialTabs.prototype.handleTabKeyboardEvent_ = function(evt) {
+      // Only react when left or right arrow pressed and tabs exist.
+      if ((evt.keyCode === this.Keycodes_.LEFT_ARROW || evt.keyCode === this.Keycodes_.RIGHT_ARROW) && this.tabsBar_) {
+        var currentIndex = Array.prototype.slice.call(this.tabs_).indexOf(evt.target);
+        // Navigate tabs in tablist with left and right arrow keys.
```

Comment:
Any a11y references to support having arrow keys behave like this for tabs? I don't recall ever seeing it before in tab systems for the web.